utils/mixins/prefixMixin.ts

```diff
@@ -18,4 +19,8 @@ export default class PrefixMixin extends Vue {
   get isMoonriver(): boolean {
     return this.urlPrefix === 'movr'
   }
+
+  get tokenId() {
+    return getKusamaAssetId(this.urlPrefix)
```

Comment:
`getKusamaAssetId` should handle also other chains

Reply:
For now, the `CommonTokenMoney ` could handle both `rmrk\bsx\snek` chains. What else chain should I add? 👀

Reply:
in the future there will be movr/glmr and astar

Reply:
Yup, but I'm not sure which token should be used in these chains.

I guess the asset list query should also add more tokens in the future. Or if I have miss something?
<img width="455" alt="image" src="https://user-images.githubusercontent.com/31397967/196952381-eec704ae-d2c4-4496-95fa-c36fc47e56be.png">

Reply:
return zero for now please

Reply:
OK, I will check.